blackbox/docs/sql/ddl/alter_table.txt

```diff
@@ -54,3 +54,26 @@ And now a nested column named ``name`` is added to the ``obj_column``::
     | obj_column['name'] | string    |
     +--------------------+-----------+
     SELECT 3 rows in set (... sec)
+
+Closing and Opening Tables
+--------------------------
+
+A table can be closed by using ``ALTER TABLE`` with the ``CLOSE`` clause::
+
+    cr> alter table my_table1 close;
+    ALTER OK, -1 rows affected (... sec)
+
+Closing a table will cause all ``SELECT``, ``INSERT`` and ``ALTER`` operations
+(apart from `OPEN`) to fail on the table, effectively forbidding reading and
+writing.
```

Comment:
remove this, again, all will fail, not only reading and writing but also altering (beside of OPEN)